liveRecorder/Tasks/ConcatinationTask.py

```diff
@@ -51,7 +53,7 @@ def find_source(self):
 
         self.logger.info("Got Bandwidths url pairs %s, find the source with the bandwidth [%s] url: [%s]",
                              str(flavor_list), maxbandwidth, maxbandwidth_url)
-        return maxbandwidth_url
+        return flavor_list
```

Comment:
need to rename the function name from find_source to extract_flavor_list . +  remove the maxbandwidth code